bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/link/LinkHandler.java

```diff
@@ -80,7 +80,7 @@
      *
      * @return Link may be invalid, but is never null
      */
-    public @NotNull Link getLink(@NotNull Resource resource, String linkURLPropertyName) {
+    public Optional<Link<Page>> getLink(@NotNull Resource resource, String linkURLPropertyName) {
```

Comment:
it could be that the reference is not a Page: shouldn't <Page> be dropped?

Reply:
Removed in dd1fd25